frontend/src/components/TopBar/RealmSelect.tsx

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR) and CISPA Helmholtz Center for Information Security
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import {FormControl, InputLabel, Select, OutlinedInput, MenuItem} from '@mui/material';
```

Comment:
To ensure the bundler doesn't include unused parts of the MUI library we sadly need to import each component individually.

```suggestion
import FormControl from '@mui/material/FormControl';
import InputLabel from '@mui/material/InputLabel';
import Select from '@mui/material/Select';
import OutlinedInput from '@mui/material/OutlinedInput';
import MenuItem from '@mui/material/MenuItem';
```